src/cdn_definitions/api/__init__.py

```diff
@@ -0,0 +1,14 @@
+import os
```

Comment:
There shouldn't be an "api" module, the API is provided from cdn_definitions module.  No need for an additional "api" layer.

The convention is that everything which is accessible underneath the top-level module, and which doesn't have a name beginning with "_", is public API.

You should stick this code somewhere under _impl, and then in cdn_definitions/\_\_init\_\_.py you should import it - that'll make it public API without exposing anything which should not be exposed.